client/me/notification-settings/blogs-settings/header.jsx

```diff
@@ -39,23 +38,35 @@ class BlogSettingsHeader extends PureComponent {
 
 	getLegend = () => {
 		const { settings } = this.props;
-		const filteredSettings = omit( settings, [
-			'blog_id',
-			'devices',
-			'email.achievement',
-			'email.store_order',
-			'email.scheduled_publicize',
-			'timeline.store_order',
-		] );
+		const {
+			blog_id,
+			devices,
+			'email.achievement': emailAchievement,
+			'email.store_order': emailStoreOrder,
+			'email.scheduled_publicize': emailScheduledPublicize,
+			'timeline.store_order': timelineStoreOrder,
+			...filteredSettings
+		} = settings;
 		// Ignore the device_id of each device found.
-		const devicesSettings = map( settings.devices, ( device ) => omit( device, 'device_id' ) );
-		const { true: onCount, false: offCount } = countBy(
-			// Here we're flattening the values of both sets of settings
-			// as both sets have two 'streams' of settings: 'email' and 'timeline'
-			[
-				...flatten( map( filteredSettings, values ) ),
-				...flatten( map( devicesSettings, values ) ),
-			]
+		const devicesSettings = Object.values( settings.devices ).map( ( device ) => {
```

Comment:
Doesn't matter that much, but we did extract `devices` above, so we could use that directly here

Reply:
Good point, updated in 687fbe71f092ad04d77b91b56ad7c99dd20cbe7c